core/src/include/Xios.hpp

```diff
@@ -28,34 +28,91 @@ class Xios : public Configured<Xios> {
     Xios();
     ~Xios();
 
-    void finalize();
+    void close_context_definition();
     void context_finalize();
+    void finalize();
     bool isInitialized();
 
     void configure() override;
     void configureServer();
-    void configureCalendar();
+    void configureCalendar(); // TODO?
+
+    /* Date and duration */
+    std::string convertXiosDatetimeToString(cxios_date datetime, bool isoFormat = true);
+    void printCXiosDate(cxios_date date);
+    void printCXiosDuration(cxios_duration duration);
 
+    /* Calendar */
     cxios_date getCalendarOrigin();
     cxios_date getCalendarStart();
-    xios::CDate getCurrentDate();
     cxios_duration getCalendarTimestep();
+    void getCalendarConfiguration(); // TODO?
```

Comment:
@TomMelt similarly here

Reply:
Again, there's no specific plan. Could possibly be used to group the calendar specific variables together perhaps. Like time_origin, start_date. But I think it would ruin the generality so I guess I never added it.

Again, feel free to delete it or keep it if you think it will be useful.

Reply:
Addressed in acc3b554.